src/libraries/Microsoft.Extensions.Logging.Console/src/Microsoft.Extensions.Logging.Console.csproj

```diff
@@ -18,14 +19,43 @@
              Link="Common\src\Extensions\Logging\NullExternalScopeProvider.cs" />
     <Compile Include="$(CommonPath)Extensions\Logging\NullScope.cs"
              Link="Common\src\Extensions\Logging\NullScope.cs" />
+    <Compile Include="$(CommonPath)System\Text\Json\PooledByteBufferWriter.cs"
+             Link="Common\System\Text\Json\PooledByteBufferWriter.cs" />
   </ItemGroup>
 
   <ItemGroup>
+    <Compile Include="$(CommonPath)Interop\Windows\Interop.Libraries.cs"
+             Link="Common\Interop\Windows\Interop.Libraries.cs" />
+    <Compile Include="$(CommonPath)Interop\Windows\Kernel32\Interop.GetConsoleMode.cs"
+             Link="Common\Interop\Windows\Interop.GetConsoleMode.cs" />
+    <Compile Include="$(CommonPath)Interop\Windows\Kernel32\Interop.GetStdHandle.cs"
+             Link="Common\Interop\Windows\Interop.GetStdHandle.cs" />
+  </ItemGroup>
+
+  <ItemGroup Condition="'$(TargetFramework)' != '$(NetCoreAppCurrent)'">
+    <Compile Include="$(CommonPath)System\Runtime\InteropServices\SuppressGCTransitionAttribute.internal.cs"
+             Link="Common\System\Runtime\InteropServices\SuppressGCTransitionAttribute.internal.cs" />
+  </ItemGroup>
+
+  <ItemGroup Condition="'$(TargetFramework)' != '$(NetCoreAppCurrent)'">
+    <Reference Include="System.Buffers" />
+    <Reference Include="Microsoft.Bcl.AsyncInterfaces" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <Reference Include="System.Collections" />
```

Comment:
Note that I have merged my PR and I think that happened after your CI run for allConfigurations started so the new code you are adding here will very likely break the build. In order to fix it, simply go to this itemgroup:

[https://github.com/dotnet/runtime/blob/b89566689612a36e0c44e55597677dd230c4f98a/src/libraries/Microsoft.Extensions.Logging.Console/src/Microsoft.Extensions.Logging.Console.csproj#L36-L42](https://github.com/dotnet/runtime/blob/b89566689612a36e0c44e55597677dd230c4f98a/src/libraries/Microsoft.Extensions.Logging.Console/src/Microsoft.Extensions.Logging.Console.csproj#L36-L42)

And make sure you add a reference to System.Core as you will need it to access IEnumerable extensions.

Reply:
I actually took the liberty of rebasing this myself and pushing the fix directly into your PR so that CI can run overnight and you still get a chance to get a green run for tomorrow morning, I hope that was ok. Beause the change was on lines that I added I did have to rebase your whole PR over latest master, I hope that was ok too.